src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Power-Strings.json

```diff
@@ -33,6 +33,13 @@
       "Mmmm... Escargot"
     ]
   },
+  "stsjorbsmod:CatharsisPower": {
+    "NAME": "Catharsis",
+    "DESCRIPTIONS": [
+      "You can play Curses. Whenever you do, lose #b%3$s HP and deal #b%1$s damage to ALL enemies for each Curse played this combat ( #b%2$s Curses played).",
+      "You can play Curses. Whenever you do, lose #b%3$s HP and deal #b%1$s damage to ALL enemies for each Curse played this combat ( #b%2$s Curse played)."
```

Comment:
IMO, the power description should more closely match Blue Candle's "Unplayable Curse cards can now be played. Whenever you play a Curse, lose # hp and Exhaust it".

Reply:
I agree that it should note the "unplayable" part; I don't think it should affect Pride, in particular